packages/lib-user/src/components/GroupStats/GroupStats.js

```diff
@@ -46,6 +46,19 @@ function GroupStats({
 
   const size = useContext(ResponsiveContext)
 
+  useEffect(function handleDefaultStartDate() {
+    // set default start date to group created_at if not defined
+    if (!selectedDateRange.startDate && group?.created_at) {
+      setSelectedDateRange({
+        endDate: selectedDateRange.endDate,
+        startDate: group.created_at.substring(0, 10)
+      })
+    }
+  }, [
+    group?.created_at,
+    selectedDateRange.startDate,
+  ])
```

Comment:
My understanding is that `startDate` is undefined on request of the url `groups/[groupId]` when the url doesn't contain the `start_date` search param. Is that correct and the scenario this `useEffect` is intended to patch? app-root doesn't know anything about the `group` resource other than its id, so its `created_at` property can't be accessed until this lib-user component?